com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -78,6 +79,11 @@ public static function getDepartmentsList($contractID)  {
               INNER JOIN civicrm_option_group cog
                 ON cov.option_group_id = cog.id
               WHERE hrjr.job_contract_id = %1
+                AND hrjr.start_date <= '$today'
+                AND (
+                  hrjr.end_date = 0
```

Comment:
i tried that, it doesn't work. the field in the table has the setting NULL = no, the default value is a 0 date '0000-00-00 00:00:00'